src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -1025,7 +1025,7 @@ $(document).ready(() => {
 
             // disable button to prevent user from clicking submission button again
             const $submissionButton = $('#response_submit_button');
-            addLoadingIndicator($submissionButton, '');
+            addLoadingIndicator($submissionButton, 'Submitting ');
```

Comment:
@whipermr5  Here is a screenshot. 
<img width="1440" alt="screen shot 2017-08-09 at 10 39 44 am" src="https://user-images.githubusercontent.com/5512789/29135588-7bf2c210-7cef-11e7-9d17-53e83f221c2b.png">